mappings/net/minecraft/world/border/WorldBorder.mapping

```diff
@@ -9,6 +9,8 @@ CLASS net/minecraft/class_2784 net/minecraft/world/border/WorldBorder
 	FIELD field_12737 centerZ D
 	FIELD field_12738 centerX D
 	FIELD field_24122 DEFAULT_BORDER Lnet/minecraft/class_2784$class_5200;
+	FIELD field_33643 STATIC_AREA_SIZE D
+	FIELD field_36191 MAX_RADIUS D
```

Comment:
`MAX_RADIUS` is probably misleading since this is not the maximum size of a border. It's the maximum of absolute value of center coordinates.

Reply:
```suggestion
	FIELD field_36191 MAX_CENTER_COORDINATES D
```